### ISSUE TITLE
Linux v 

### ISSUE THREAD

**Issue by KimB2017** [sillsdev/ptx2pdf#356]
Today's version of ptxprint seems to have a syntax error.
This morning I saw 1.5.7-0~... (see Synaptic log below), which failed to install. This afternoon I saw 1.5.8-0~. It also failed with a very similar-looking error.

==============================
Terminal trace on startup after it failed to install:

```
ThinkPad-E495:~$ ptxprint
Traceback (most recent call last):
  File "/usr/bin/ptxprint", line 16, in <module>
    from ptxprint.view import ViewModel, VersionStr, doError
  File "/usr/lib/python3/dist-packages/ptxprint/view.py", line 9, in <module>
    from ptxprint.styleditor import StyleEditor
  File "/usr/lib/python3/dist-packages/ptxprint/styleditor.py", line 43
    if key in self.sheet.get(mrk, {}) and 
                                         ^
SyntaxError: invalid syntax
```
==============================
Synaptic log trace:

```
Unpacking python3-ptxprint (1.5.8-0~202101271741~ubuntu20.04.1) over (1.5.7-0~202101271032~ubuntu20.04.1) ...
Setting up python3-ptxprint (1.5.8-0~202101271741~ubuntu20.04.1) ...
Traceback (most recent call last):
  File "/usr/lib/python3.8/py_compile.py", line 144, in compile
    code = loader.source_to_code(source_bytes, dfile or file,
  File "<frozen importlib._bootstrap_external>", line 846, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3/dist-packages/ptxprint/styleditor.py", line 43
    if key in self.sheet.get(mrk, {}) and 
                                         ^
SyntaxError: invalid syntax

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/py_compile.py", line 197, in main
    compile(filename, doraise=True)
  File "/usr/lib/python3.8/py_compile.py", line 150, in compile
    raise py_exc
__main__.PyCompileError:   File "/usr/lib/python3/dist-packages/ptxprint/styleditor.py", line 43
    if key in self.sheet.get(mrk, {}) and
                                         ^
SyntaxError: invalid syntax


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.8/py_compile.py", line 218, in <module>
    sys.exit(main())
  File "/usr/lib/python3.8/py_compile.py", line 200, in main
    if quiet < 2:
NameError: name 'quiet' is not defined
dpkg: error processing package python3-ptxprint (--configure):
 installed python3-ptxprint package post-installation script subprocess returned error exit status 1
Processing triggers for desktop-file-utils (0.24-1ubuntu3) ...
Processing triggers for mime-support (3.64ubuntu1) ...
Processing triggers for hicolor-icon-theme (0.17-2) ...
Processing triggers for gnome-menus (3.36.0-1ubuntu1) ...
Processing triggers for bamfdaemon (0.5.3+18.04.20180207.2-0ubuntu2) ...
Rebuilding /usr/share/applications/bamf-2.index...
Errors were encountered while processing:
 python3-ptxprint
E: Sub-process /usr/bin/dpkg returned an error code (1)
A package failed to install.  Trying to recover:
Setting up python3-ptxprint (1.5.8-0~202101271741~ubuntu20.04.1) ...
Traceback (most recent call last):
  File "/usr/lib/python3.8/py_compile.py", line 144, in compile
    code = loader.source_to_code(source_bytes, dfile or file,
  File "<frozen importlib._bootstrap_external>", line 846, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3/dist-packages/ptxprint/styleditor.py", line 43
    if key in self.sheet.get(mrk, {}) and 
                                         ^
SyntaxError: invalid syntax

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/py_compile.py", line 197, in main
    compile(filename, doraise=True)
  File "/usr/lib/python3.8/py_compile.py", line 150, in compile
    raise py_exc
__main__.PyCompileError:   File "/usr/lib/python3/dist-packages/ptxprint/styleditor.py", line 43
    if key in self.sheet.get(mrk, {}) and
                                         ^
SyntaxError: invalid syntax


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.8/py_compile.py", line 218, in <module>
    sys.exit(main())
  File "/usr/lib/python3.8/py_compile.py", line 200, in main
    if quiet < 2:
NameError: name 'quiet' is not defined
dpkg: error processing package python3-ptxprint (--configure):
 installed python3-ptxprint package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 python3-ptxprint
```
=======================================================

**Comment by markpenny** [comment]
@KimB2017 Hi Kim, we're sorry about that. It was a simple syntax error introduced late last night. Please look out for **version 1.5.8.1** which should include the fix. Hopefully it will show up soon - like an hour or so. I'm not sure where you're pulling your Linux packages from, but MH pointed me to [this page](https://launchpad.net/~silnrsi/+archive/ubuntu/ptxprint/+packages) a few days ago and it seems to have the latest.

**Comment by KimB2017** [comment]
Thanks, Mark, I figured it was "a simple syntax error." I'll fetch the 
new one in the morning.

FYI, I have the ptxprint PPA (repository) enabled, which gives me the 
latest version, along with most Ubuntu/Wasta system and program updates, 
when I ask it to check.

*Kim Blewett*
Language Technology Consultant (LSS)
SIL Papua New Guinea

**Comment by KimB2017** [comment]
Well... turns out I had already tried (from the PPA) 1.5.8-0~202101271741~ubuntu18.04.1 yesterday, before I created this issue. It gives me the same error as the previous version. I tried uninstalling, re-downloading, rebooting, etc., in case the failed install placed a file in there, but still no success.

Is anyone else testing the Linux version? I know that recently a few people have learned about it and installed it, but I don't know if they are updating daily to the newest version.
